libafl/src/inputs/bytes.rs

```diff
@@ -11,7 +11,7 @@ use core::cell::RefCell;
 use libafl_bolts::{ownedref::OwnedSlice, HasLen};
 
 use super::ValueInput;
-use crate::inputs::{HasMutatorBytes, HasTargetBytes};
+use crate::inputs::{HasMutatorBytes, HasMutatorResizableBytes, HasTargetBytes};
```

Comment:
I'd prefer to have `HasFixedMutatorBytes` since the default is that they are resizable(?)
Otherwise at least `HasResizableMutatorBytes`